CONTRIBUTING.md

```diff
@@ -1,6 +1,31 @@
 ## How to update Gems when some Gems are vulnerable
 
-- Update ruby to the latest version in [.tool-versions](./tool-versions)
-- Update Bundler to the latest version with `gem install bundler`
-- Use the latest version of Bundler to manage Gemfile.lock `bundle update --bundler`.
-- Update Gems with `bundle update`.
+```
+# Run bash shell, if you are not using it already
+bash 
+
+# Clean PATH
+export PATH=""
+
+# Initialize PATH
+. /etc/profile
+
+# Bring in homebrew
+eval "$(/opt/homebrew/bin/brew shellenv)"
+
+# Assume you have install libyaml with homebrew, as ruby depends on it.
+
+# Bring in asdf
+. "$HOME"/.asdf/asdf.sh
```

Comment:
Why these steps are needed?

Reply:
If some entries in your PATH shadow the command used by ruby-build, like `nm`, then `asdf install` will fail. These steps ensure the shell is in a known clean state, such that `asdf install` (that is, ruby-build) can compile successfully.